### PR TITLE
Edits to text in Index API doc

### PR DESCRIPTION
**Pull request by dmeiss**
Minor edits to punctuation, capitalization, formatting, and word usage.

### CODE REVIEW

**Review by EmmanuelDemey — APPROVED** [review]

File: docs/reference/docs/index_.asciidoc [hunk 1]
```diff
@@ -293,22 +293,22 @@ replication succeeded/failed.
 === Refresh
 
 Control when the changes made by this request are visible to search. See
-<<docs-refresh,refresh>>.
+<<docs-refresh,`?refresh`>>.
```
Comment (danielmitterdorfer) [inline docs/reference/docs/index_.asciidoc]: I think we should stick with the existing reference here. To me especially the leading question mark is confusing.

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

**Comment by dmeiss** [comment]
Switched back the edits to the refresh cross-reference (removed the question mark and Code styling)

**Review by danielmitterdorfer — APPROVED** [review]
Thanks for iterating! Looks good to me now.

**Pull request merged**